projects/toros/index.js

```diff
@@ -14,10 +14,11 @@ async function tvl(api) {
   const poolSummaries = await api.multiCall({
     abi: TOROS_POOL_ABI,
     calls: pools,
+    permitFailure: true
   });
 
   const totalValue = poolSummaries.reduce(
-    (acc, i) => acc + +i.totalFundValue,
+    (acc, i) => i && acc + +i.totalFundValue,
```

Comment:
this could lead to NaN, safer to filter the array before 
![image](https://github.com/user-attachments/assets/4aa4e62e-2d5d-44a9-8035-dbdfbcb4d246)


also, any idea why it fails sometimes?